spec/factories/access_control_entry.rb

```diff
@@ -28,5 +28,10 @@
         ]
       end
     end
+
+    after :create do |access_control_entry|
+      access_control_entry.aceable ||= create(:portfolio)
```

Comment:
@eclarizio I favored this approach as it lessens the need to modify several existing specs.